### PR TITLE
[14.0][MIG] base_location_nuts: Migration to 14.0

### PR DESCRIPTION
**Pull request by kos94ok-3D**

### CODE REVIEW

**Comment by kos94ok-3D** [comment]
@tecnativa Could you review it?

**Comment by pedrobaeza** [comment]
Please don't ping `@Tecnativa`, as that's not a real user and it's a key we use internally, altering our procedure. It's good that if you ask for a review, do a couple or 3 of other reviews in exchange.

**Comment by kos94ok-3D** [comment]
OK. Thanks for your response.

File: base_location_nuts/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_res_partner_nuts_user_all","res_partner_nuts group_partner_manager_all","model_res_partner_nuts","",1,0,0,0
 "access_res_partner_nuts_user","res_partner_nuts group_partner_manager","model_res_partner_nuts","base.group_partner_manager",1,1,1,1
+"access_nuts_import","nuts.import access","model_nuts_import","base.group_user",1,1,1,1
```
Comment (pedrobaeza) [inline base_location_nuts/security/ir.model.access.csv]: I don't think all users access to this wizard. Can you restrict this a bit?

Comment (kos94ok-3D) [reply base_location_nuts/security/ir.model.access.csv]: for `group_partner_manager`?

Comment (kos94ok-3D) [reply base_location_nuts/security/ir.model.access.csv]: Done and update Readme

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**